scripts/sync-postgres-exporter-mixin.sh

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+exporter="tmp/postgres_exporter"
+mixin="${exporter}/postgres_mixin"
+
+cd "$(git rev-parse --show-toplevel)"
+
+if [ -d "$exporter" ]; then
+	rm -fr "$exporter"
+fi
+mkdir -p tmp
+git clone --depth 1 https://github.com/prometheus-community/postgres_exporter.git tmp/postgres_exporter
+
+# This mixin requires a few tools to be installed, we need to check for those
+# and print out that they need to be installed if they are not.
+
+if [ ! -x "$(command -v go)" ]; then
+	echo "go is not installed, please install it"
+	exit 0
```

Comment:
```suggestion
	exit 1
```
we shouldn't exit with success status code when the script cannot be executed properly